warehouse/query-core/src/main/java/datawave/query/jexl/functions/QueryFunctionsDescriptor.java

```diff
@@ -191,8 +191,12 @@ private static void verify(String name, int numArgs) {
                 break;
             case QueryFunctions.UNIQUE_FUNCTION:
             case QueryFunctions.UNIQUE_BY_DAY_FUNCTION:
-            case QueryFunctions.UNIQUE_BY_HOUR_FUNCTION:
+            case QueryFunctions.UNIQUE_BY_YEAR_FUNCTION:
```

Comment:
We lost `UNIQUE_BY_HOUR` here. Do we want to keep that?

Reply:
Good catch, will put that back in.